### ISSUE TITLE
Add main chain and tlbc node to bridge docker compose file

### ISSUE THREAD

**Issue by berndbohmeier** [trustlines-protocol/blockchain#109]

**Comment by berndbohmeier** [comment]
Depends on #108

**Comment by saschagoebel** [comment]
Closed as this belongs to the PoA bridge. We'll create a new task if needed for the new bridge.
